pkg/database/database.go

```diff
@@ -210,10 +210,7 @@ func (db *Database) performResourceQuery(ctx context.Context, pq *paramQuery) ([
 	lastRow := parsedRows[len(parsedRows)-1]
 
 	for _, parsedRow := range parsedRows {
-		r, errConversion := models.UnstructuredFromByteSlice(parsedRow.Resource)
-		if errConversion != nil {
-			return resources, 0, "", errConversion
-		}
+		r := string(parsedRow.Resource)
```

Comment:
If parsing the `data` field is just going to be casting it to string I would suggest removing the whole `for` block and change the the retrieved type in the struct from `[]byte` to `string` (line 196)

Reply:
How do you go about removing the for loop? I need to loop over the rows to append their `Resource` to the list of resources.

Reply:
Right, the for is needed, but you get rid of the parsing operation :) 
LGTM